contracts/managers/fee/FeeManager.sol

```diff
@@ -483,6 +483,29 @@ contract FeeManager is AccessManagedUpgradeable, ContextClient {
         FeeManagerStorageLib.updateIntervalHighWaterMarkPerformanceFee(updateInterval_);
     }
 
+    /// @notice Sets the deposit fee percentage for the plasma vault
+    /// @dev This function configures the deposit fee that will be charged on all new deposits
+    ///      to the plasma vault. The fee is calculated as a percentage of the deposited assets
+    ///      and is deducted from the user's deposit amount before minting shares.
+    ///
+    /// @param fee_ Deposit fee percentage with 18 decimal precision (1e18 = 100%, 1e16 = 1%)
+    ///              Must be less than 1e18 (100%) to prevent total loss of deposited assets
+    ///
+    /// @dev Flow:
+    /// 1. Validates that the fee is within acceptable bounds (should be < 100%)
+    /// 2. Updates the deposit fee in storage via FeeManagerStorageLib
+    /// 3. The new fee will apply to all subsequent deposits
+    ///
+    /// @custom:access Restricted to authorized roles (ATOMIST_ROLE)
+    /// @custom:security Fee should be reasonable to maintain vault attractiveness
+    /// @custom:security Fee is deducted from user deposits, not added on top
+    /// @custom:see EIP-4626 for vault fee patterns and best practices
+    /// @custom:example Setting fee_ to 1e16 (1e18 * 0.01) sets a 1% deposit fee
+    function setDepositFee(uint256 fee_) external restricted {
```

Comment:
Change everywhere name Deposit Fee to Onboarding Contribution - don't use fee but onboardingContributionValue or contributionValue